WNPRC_EHR/resources/queries/study/damSireSuffixMatcher/.qview.xml

```diff
@@ -0,0 +1,11 @@
+<customView xmlns="http://labkey.org/data/xml/queryCustomView">
+    <columns>
+        <column name="last_four"/>
+        <column name="sire"/>
+        <column name="dam"/>
+        <column name="frequency"/>
```

Comment:
Should we include the species? that way you can ignore results were the species are different.